test/performance/benchmarks/rollout-probe/rollout-probe-setup-activator-direct-lin.yaml

```diff
@@ -33,7 +33,7 @@ spec:
             cpu: 20m
             memory: 20Mi
           limits:
-            cpu: 50m
-            memory: 50Mi
-      containerConcurrency: 5
+            cpu: 200m
```

Comment:
The probe tests were failing also downstream and initially I thought the delay was due to ingress/activator's limited capacity but no change made them pass. The delay seems to come from the QP side so being more generous here.